ResearchUI/ResearchUI/iOS/Views/RSDCheckmarkView.swift

```diff
@@ -146,7 +150,7 @@ fileprivate let defaultSize: CGFloat = 122
         _shapeLayer.frame = self.layer.bounds
         self.layer.cornerRadius = _cornerRadius ?? 0
         
-        _backgroundLayer.frame = self.layer.frame.insetBy(dx: self.backgroundInset, dy: self.backgroundInset)
```

Comment:
self.layer.frame actually references the frame within this view's superview.  So in the case of both triggers and medication logging, the x: and y: were way off.

Reply:
Can you please change to
:
```
_backgroundLayer.frame = self.layer.bounds.insetBy(dx: self.backgroundInset, dy: self.backgroundInset)
```

Thanks, and thanks for fixing. :)